### ISSUE TITLE
It doesn't work on the lib umqtt from micropython

### ISSUE THREAD

**Issue by kevinkm** [martin-ger/uMQTTBroker#33]
it works well, however, I try to use umqtt lib to connect this broker, I don't think they both know each other, anyway hope Martin can fix the bug asap.

**Comment by kevinkm** [comment]
It's weird, I just try to turn on the AP mode and I connect to the AP from esp8266, I set the broker server as same as the gateway 192.168.4.1, since also the monitor tells "AP started
, IP address: 192.168.4.1, Starting MQTT broker " , but the mqtt client can't connect the mqtt server. However, I start this esp8266 without AP mode and connect a router, and my mqtt client connected a same router , I set broker server IP just like esp8266's IP in the router, all of sudden, the mqtt client just connected the mqtt server. Jesus, why?

Oh, when I use two iPhones connected the esp8266 with AP mode, I've tested with MQTTool, it's fine.

So I believe this is a big bug.

**Comment by kevinkm** [comment]
Figure out! No problem now!

**Comment by martin-ger** [comment]
So you have solved it?

**Comment by kevinkm** [comment]
Yeah, problem solved, anyway I think it's a little compatible issue between micropython and arduino, your code is working fine, thanks Martin.
